std/json.d

```diff
@@ -359,6 +359,90 @@ struct JSONValue
         return type == JSONType.null_;
     }
 
+    /***
+     * Generic type value getter
+     * A convenience getter that returns `JSONValue` as the specified D type.
+     * Note: only arithmetic, `bool`, `string`, `JSONValue` and `JSONValue[]` types are accepted
+     * Throws: `JSONException` if `T` is not a valid type or conversion is impossible
```

Comment:
More avoiding using the "conversion" term: (Also, in some cases, conversion is "possible", we just refuse to do it implicitly)
```suggestion
     * Throws: `JSONException` if `T` cannot hold the contents of this `JSONValue`
```